src/server/web_server.js

```diff
@@ -149,17 +155,21 @@ app.use(express_compress());
 ////////////
 // ROUTES //
 ////////////
-if (prom_reports.instance().enabled()) {
+if (prom_reporting.is_enabled()) {
+    // Enable proxying for all metrics servers
+    app.use('/metrics/web_server', express_proxy(`localhost:${config.WS_METRICS_SERVER_PORT}`));
```

Comment:
1. How does this appear in Prometheus ? Any diff between a the same metric exported here and via /metrics ?
2. Isn't /metric the default path Prometheus is scraping ? will it scape by default /metrics/web_server ?
3. What happens when both routes are scraped (old and new) duplication ? override ? (in prometheus)

Reply:
currently, prom is only the old path this is why I still support it.
After I will push a PR to noobaa-operator I will get rid of the old paths

Reply:
Have you verified we can change paths ? I mean besides the podmonitor which you have tested ...

Reply:
yes. checked that it is possible. It is an edit on the service monitor CR. And it supports multiple scrape paths